Documentation/troubleshooting.rst

```diff
@@ -137,15 +142,53 @@ e.g.:
 Connectivity Problems
 =====================
 
+Cilium connectivity tests
+------------------------------------
+
+The Cilium connectivity test_ deploys a series of services, deployments, and 
+CiliumNetworkPolicy which will use various connectivity paths to connect to 
+each other. Connectivity paths include with and without service load-balancing 
+and various network policy combinations. The pod name indicates the connectivity
+variant and the readiness and liveness gate indicates success or failure of the
+test:
+
+.. _test: https://raw.githubusercontent.com/cilium/cilium/1.7.4/examples/kubernetes/connectivity-check/connectivity-check.yaml
```

Comment:
i think you need to use the `SCM_WEB` variable defined in Documentation/conf.py here to avoid hardcoding branches/tags. here is an example: https://github.com/cilium/cilium/blame/054dd163fdb792c44b3c493b8d6fd111e5e1a2bc/Documentation/gettingstarted/dns.rst#L33